src/libcore/future/future.rs

```diff
@@ -23,7 +23,7 @@ use crate::task::{Context, Poll};
 /// When using a future, you generally won't call `poll` directly, but instead
 /// `await!` the value.
 #[doc(spotlight)]
-#[must_use = "futures do nothing unless polled"]
+#[must_use = "futures stay idle unless you await them"]
```

Comment:
```suggestion
#[must_use = "futures stay idle unless you `.await` them"]
```
To hint at the surface syntax :)

Reply:
Thanks :)